api/base/views.py

```diff
@@ -560,8 +560,6 @@ class BaseNodeLinksDetail(JSONAPIBaseView, generics.RetrieveAPIView):
 
 class BaseNodeLinksList(JSONAPIBaseView, generics.ListAPIView):
 
-    ordering = ('-modified',)
```

Comment:
This was breaking things by referencing the model field name instead of the serializer.